components/build-service/development/kustomization.yaml

```diff
@@ -2,6 +2,17 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - ../base
+- https://github.com/redhat-appstudio/build-service/config/default?ref=45e8fe7c3b036d1df3eee11c1d83e0633e451c07
+
+namespace: build-service
+
+images:
+- name: quay.io/redhat-appstudio/build-service
+  newName: quay.io/redhat-appstudio/build-service
+  newTag: 45e8fe7c3b036d1df3eee11c1d83e0633e451c07
+
+commonAnnotations:
+  argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

Comment:
Does `commonAnnotations` need to be here as well, or is it enough that it's already in `base/kustomization.yaml`?

Reply:
we need to add it here as well, since resource `https://github.com/redhat-appstudio/build-service/config/default?ref=45e8fe7c3b036d1df3eee11c1d83e0633e451c07` is being moved to individual overlays